docs/metadata.md

```diff
@@ -11,16 +14,80 @@ How To:
 The metadata collection is done through the use of [Stockpile](https://github.com/cloud-bulldozer/stockpile), [Backpack](https://github.com/cloud-bulldozer/backpack) and [Scribe](https://github.com/cloud-bulldozer/scribe).
 The data is then uploaded to a defined Elasticsearch instance.
 
-When launching your benchmark, if enabled, the metadata collection container (backpack)
-will launch as a DaemonSet on all nodes in the cluster. It will then run stockpile
-to gather data and then push that information up to Elasticsearch.
+There are two ways to launch metadata collection from a workload.
+
+# Targeted Init Containers
```

Comment:
Can you please update docs to reflect that ycsb doesn't support this ? And maybe also add a note to ycsb docs?

Reply:
ack. good call

Reply:
updated